nsxt/resource_nsxt_upgrade_run.go

```diff
@@ -564,48 +561,23 @@ func updateUpgradeUnitGroups(upgradeClientSet *upgradeClientSet, d *schema.Resou
 				typeHost := "HOST"
 				groupGet = &model.UpgradeUnitGroup{DisplayName: &groupName, UpgradeUnits: groupMembers, Type_: &typeHost}
 			} else {
-				// This custom group might be updated, compare the upgrade unit lists
-				nsxUnits := getUnitIDsFromUnits(groupGet.UpgradeUnits)
-
-				// Find and remove upgrade units which have been removed from the list. This is done by assigning the
-				// upgrade unit to its default group
-				var schemaUnits []string
-				if group["hosts"] != nil {
-					schemaUnits = interface2StringList(group["hosts"].([]interface{}))
-				}
-				getHostDefaultUpgradeGroup, err := getHostDefaultUpgradeGroupGetter(m, groupList)
+				return fmt.Errorf("upgrade unit group with name %s already exists", groupName)
```

Comment:
A generic concern: why are we now failing in this case, and why don't we need anymore logic for handling the case where custom group could have been updated?
It's likely simply because I do not fully understand the logic here.

Reply:
Got it, thanks!